src/material-experimental/mdc-form-field/directives/floating-label.ts

```diff
@@ -25,16 +25,13 @@ import {ponyfill} from '@material/dom';
 @Directive({
   selector: 'label[matFormFieldFloatingLabel]',
   host: {
-    'class': 'mdc-floating-label',
-    '[class.mdc-floating-label--required]': 'required',
+    'class': 'mdc-floating-label mat-mdc-floating-label',
```

Comment:
We should add a short note on that the asterisk needs to be handled manually

Reply:
There's a note already in the `form-field.html`.

Reply:
I consider this decoupled though, but that's super picky anyway, so good with me 👍